internal/sql/repository/DockerArtifactStoreRepository.go

```diff
@@ -45,9 +45,13 @@ type DockerArtifactStore struct {
 	models.AuditLog
 }
 
-func (store *DockerArtifactStore) GetRegistryLocation() (registryLocation string) {
-	urls := strings.Split(store.RegistryURL, "//")
-	return urls[1]
+func (store *DockerArtifactStore) GetRegistryLocation() (registryLocation string, err error) {
+	u, err := url.Parse(registryLocation)
```

Comment:
earlier we were taking whole url.. now we are returning only domain... is this correct?

Reply:
we were using it for docker login, so it would work